### ISSUE TITLE
Potential for confusion between "main" and "solution" editors in embedded layout

### ISSUE THREAD

**Issue by Erhannis** [dart-lang/dart-pad#2103]
### Page URL

https://dart.dev/codelabs/dart-cheatsheet.html

### Page source

https://github.com/dart-lang/site-www/tree/master/src/codelabs/dart-cheatsheet.md

### Describe the problem

The embedded editors don't compile code - they give syntax errors, even for the provided solutions.  See screenshot:

![Failing compile](https://user-images.githubusercontent.com/3024589/140959839-1cb4ee75-630c-4bae-8a94-23ce7edc5711.png)

### Expected fix

Depending on the _cause_ - update a broken dependency, fix bug, provide missing file, etc., such that the editors successfully compile at least the provided solutions.

### Additional context

Firefox, several versions, including 94.0 .  Happens both on windows and linux, though that should be unsurprising.

**Comment by parlough** [comment]
Coming back to this issue, the multi-file embedded layout is no longer supported, so this confusion should no longer occur. https://github.com/dart-lang/dart-pad/issues/2808 has some more information.

Thanks again for reporting this and sorry about the original confusion.
